feeding-game/src/game/game.js

```diff
@@ -366,7 +369,7 @@ function Game() {
 
     // Button to go to campaign
     var px = game.world.centerX;
-    var py = pitch.position.y + margin * 6;
+    var py = pitch.position.y + margin * 6; //@todo possibly lower to make room for pitch text
```

Comment:
Yea, I'm down for lowering it if it's not getting in the way of other things below. Wherever it does get placed, it should look good whether the pitch text is 1, 2 or 3 lines large.